.github/workflows/ci.yml

```diff
@@ -0,0 +1,101 @@
+name: Django Rest Framework Reactive CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+*"
+
```

Comment:
Add workflow_dispatch trigger, so it can be run manually if necessary.

Reply:
OK.